### PR TITLE
UGENE-5660. Crash when calculating distance matrix for big alignment

### PR DESCRIPTION
**Pull request by yalgaer**
The patch contains only fix and I will add GUI test before merging it into master

### CODE REVIEW

File: src/corelibs/U2Core/src/util/FileAndDirectoryUtils.cpp [hunk 1]
```diff
@@ -127,13 +128,21 @@ bool FileAndDirectoryUtils::isFileEmpty(const QString& url) {
...
-        return;
-    }
+    CHECK(f != nullptr && str.length() >= MIN_LENGTH_TO_WRITE, );
     f->write(str.toLocal8Bit());
     str.clear();
 }
 
+bool FileAndDirectoryUtils::storeTextToFile(const QString& filePath, const QString& text) {
+    QFile file(filePath);
+    if (!file.open(QFile::WriteOnly)) {
```
Comment (DmitriiSukhomlinov) [inline src/corelibs/U2Core/src/util/FileAndDirectoryUtils.cpp:138]: `CHECK`

**Review by DmitriiSukhomlinov — CHANGES_REQUESTED** [review]
Translations are required

**Comment by DmitriiSukhomlinov** [comment]
Also - GUI test?

**Comment by yalgaer** [comment]
> Also - GUI test?

Yes, please check the  summary

**Pull request merged**